src/celo/fees.ts

```diff
@@ -21,6 +21,7 @@ export const fees: ChainFees<typeof formatters> = {
     params: ChainEstimateFeesPerGasFnParameters<typeof formatters>,
   ) => {
     if (!params.request?.feeCurrency) return null
+    // TODO check if multiply is noe empty?
```

Comment:
there is already a check where this function is called that ensures that multiple is always passed in. so if anything only do the smallest of checks internally. https://github.com/celo-org/viem/blob/dccccc9687e948bd44510a0710f9014671ce75bd/src/actions/public/estimateFeesPerGas.ts#L126

Reply:
I was just wondering because `multiply` in function args seems like a mandatory property, but still I was able to write the tests without passing it explicitly and initially the tests failed because it was missing

Reply:
lets remvoe comment then